examples/resources/incident_alert_route/resource.tf

```diff
@@ -0,0 +1,113 @@
+resource "incident_alert_route" "service_alerts" {
+  name       = "Testing Alert Routes"
+  enabled    = true
+  is_private = false
+
+  alert_sources = [
+    {
+      alert_source_id  = incident_alert_source.http.id
+      condition_groups = []
+    }
+  ]
+
+  condition_groups = [
+    {
+      conditions = [
+        {
+          subject        = "alert.title"
+          operation      = "is_set"
+          param_bindings = []
+        }
+      ]
+    }
+  ]
+
+  expressions = []
+
+  channel_config = []
+
+  escalation_config = {
+    auto_cancel_escalations = true
+    escalation_targets = [
+      {
+        escalation_paths = {
+          array_value = [
+            {
+              literal = "01JPQNFD3RWAAY2V83QQ80D1ZV"
+            }
+          ]
+        }
+        # Users field can be included if needed
```

Comment:
I think worth explicitly calling out that each target is *either* a user *or* an escalation path - if you try to use both you'll get an error!